dask/bag/core.py

```diff
@@ -1134,7 +1134,7 @@ def partition(grouper, sequence, npartitions, p, nelements=2**20):
         d2 = defaultdict(list)
         for k, v in d.items():
             d2[abs(hash(k)) % npartitions].extend(v)
-        p.append(d2)
+        p.append(d2, fsync=True)
```

Comment:
I don't suppose you have a test for this?  If not that's fine.  I'm curious in what conditions it arose though.